content/md/supplier-onboarder/use-assets.md

```diff
@@ -0,0 +1,28 @@
+---
+id: use-assets
+themes: manage-products
+title: Use assets
+popular: false
+---
+
+# Manage assets
+Assets are now present in a new tab of the product edition page. On this page you are able to visualize all assets and also modify them.
```

Comment:
"modify them" = éditer les attributs des assets, ou bien changer les assets liés (c'est à dire, changer les liens entre assets et produits)? Si c'est la 2e option, je dirais plutôt "and also modify which assets have are linked to the product."